conjure-java-core/src/main/java/com/palantir/conjure/java/types/AliasGenerator.java

```diff
@@ -145,6 +146,16 @@ public static JavaFile generateAliasType(TypeMapper typeMapper, AliasDefinition
                     .addCode(intCastCodeBlock)
                     .build());
 
+            spec.addMethod(MethodSpec.methodBuilder("of")
+                    .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
```

Comment:
We might as well make this private

Reply:
I followed the pattern of the other factory methods. I'm not sure if making the method private will cause issues for Jackson

Reply:
Private is supported by Jackson, I don’t think we should make this public if we can help it.

At a certain point we may just want to generate deserializers...